dts/arm/nuvoton/npcx7m6fb.dtsi

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Please see previous comment.

Reply:
I still don't understand.. Would this be in the nuvoton/ directory even if we renamed the other directory to nuvoton_numicro/ ?  (if the answer is yes, then you can just resolve this comment, because I think I understand then)